releases/releases.go

```diff
@@ -0,0 +1,146 @@
+package releases
```

Comment:
This has been moved out of internal so that it is accessible by the Registry api. @TristanSpeakEasy I assume this isn't what you had in mind when you said "might be worth moving this into a pkg module in this repo so you can just import it as a dependency and don't need to worry about keeping it in sync with the action". I tried a few different ways of doing that and wasn't able to get anything going I was happy with. Let me know if you have any suggestions